README.md

```diff
@@ -5,3 +5,7 @@
 ### September 20th 2017
 
 * [Introduction](http://phpugms.github.io/phpugms_greeting_2017-09/index.html)
+
+### October 17th 2017
```

Comment:
Can you please put that above the september meetup so we have the most recent meetup at the beginning.

Reply:
fixed